### ISSUE TITLE
Melee-only mobs such as Zombies and Spiders don't do damage to player

### ISSUE THREAD

**Issue by PantsWP** [Extegral/Enigmatic-Legacy#342]
#### Versions (Be specific, do not write "latest"):
 * Enigmatic Legacy: 2.22.1
 * Curios API: 1.18.2-5.0.7.0
 * Patchouli: 1.18.2-69
 * Forge: 40.1.31

#### Description of a Bug:
Whenever encountering a mob that only deals melee damage, such as zombies, spiders and endermen, they do no damage to the player and instead will play the armor/tool breaking sound. They sometimes also get knocked back. Skeletons and creepers do not have this issue, haven't tested with other mobs. Tested on a fresh singleplayer install and world with just Enigmatic Legacy and its required dependencies. 

#### How to Reproduce:
1. Create new or join existing world
3. Encounter a mob that does direct melee damage and let it hit you
4. Behold!

No errors appear in the logs nor is there a crash so feel no need to attach one.

**Comment by Aizistral** [comment]
I am not sure you yourself have followed described steps for reproduction. What you describe is a behavior characteristic of Non-Euclidean Cube, since it has the ability to block melee attacks that somehow exceed 100 (or 150 with Ring of the Seven Curses) damage. Knocking back is another ability of the cube. But for that you need to have the cube equipped, and obviously the attack has to deal that much damage.

**Comment by StygianNights** [comment]
I’m ganna look into this when I get off work. In the meantime, if you could please provide your latest.log after attempting to reproduce the issue once more. This should tell us what other mods might be causing it, if it turns out I cannot reproduce the issue.

**Comment by PantsWP** [comment]
> I am not sure you yourself have followed described steps for reproduction. What you describe is a behavior characteristic of Non-Euclidean Cube, since it has the ability to block melee attacks that somehow exceed 100 (or 150 with Ring of the Seven Curses) damage. Knocking back is another ability of the cube. But for that you need to have the cube equipped, and obviously the attack has to deal that much damage.

When I tested it, my inventory and Curios were empty and I was in Survival. No configs have been touched. It stopped occurring when I removed the mod. So yes, I did follow those exact steps, I can post a video if you want me to.

I’ll post a latest.log when I get on my PC a little later, although I have looked at it and couldn’t find anything useful.

**Comment by StygianNights** [comment]
Please do, with a full log from the session please. I attempted to reproduce and could not with only JEI, Curios, Enigmatic, Caelus, and Forge .31

**Comment by PantsWP** [comment]
Gosh, I'm such an idiot. So basically what is happening is I am creating a modpack and started encountering this bug, so I went through the mods trying to single out the culprit. When I found out it was EL, I then created a separate instance with just EL and _Improved Mobs_ and their required dependencies, which increases the damage and health of mobs over time. I had it configured so that they would deal 50x melee damage at the highest difficulty (100), so a zombie would deal 150 damage with each hit on Normal difficulty. 

I just worked out that removing IM won't remove the attributes to mobs that were spawned prior to the removal and so when I went back to test again and spawned a new zombie, the issue didn't occur, which means it is strictly related to mobs that have their damage turned up past 100. However, my inventory and Curios slots were still empty so maybe the effects of the Cube are still occurring despite not having it equipped as I wasn't aware of its existence prior to this report. So it seems to be a compat issue with Improved Mobs (or perhaps with increasing the damage attribute in general but got no way to test that outside IM).

Still, I have posted a video anyway and have provided a `latest.log` as promised.

https://user-images.githubusercontent.com/1592219/172037060-8352f346-6674-4123-9566-1695ba45c435.mp4
[latest.log](https://github.com/Aizistral-Studios/Enigmatic-Legacy/files/8839392/latest.log)

I also used Improved Mobs v1.10.11 and TenshiLib v1.6.8 alongside the previously mentioned mods as presented in the video.
Here are the configs, just drop the improvedmobs folder into the config folder:
[improvedmobs.zip](https://github.com/Aizistral-Studios/Enigmatic-Legacy/files/8839405/improvedmobs.zip)
